README.md

```diff
@@ -275,3 +260,192 @@ datetime
+
+```python
+
+# The "/" at the end of the first argument is important. If you omit it you'll get an error.
+prices = san.get(
+    "erc20_exchange_funds_flow/",
+    from_date="2018-04-16T10:02:19Z",
+    to_date="2018-05-23T10:02:19Z"
+)
+
+```
+
+Example result:
+
+```bash
+
+                                     contract  exchangeDiff  exchangeDiffBtc  \
+0   0x006bea43baa3f7a6f765f14f10a1a1b08334ef45 -5.353089e+03        -0.691860
+1   0x0371a82e4a9d0a4312f3ee2ac9c6958512891372 -1.993464e+04        -0.050134
+2   0x08711d3b02c8758f2fb3ab4e80228418a7f8e39c  2.712031e+06       209.894542
+3   0x089a6d83282fb8988a656189f1e7a73fa6c1cac2  1.214960e+04         0.000000
+4   0x08f5a9235b08173b7569f83645d2c7fb55e8ccd8 -9.398656e+05        -1.687275
+
+    exchangeDiffUsd    exchangeIn  exchangeInBtc  exchangeInUsd   exchangeOut  \
+0     -7.017060e+03  4.213794e+04       2.226106   1.999951e+04  4.749103e+04
+1     -4.372710e+02  3.120267e+04       0.078270   7.055826e+02  5.113730e+04
+2      1.897489e+06  5.308479e+06     397.788044   3.584974e+06  2.596448e+06
+3      0.000000e+00  6.740607e+04       0.000000   0.000000e+00  5.525647e+04
+4     -2.628859e+03  3.067355e+07     412.468640   3.711345e+06  3.161342e+07
+
+    exchangeOutBtc  exchangeOutUsd  exchangeVolumeBtc  exchangeVolumeUsd  \
+0         2.917965    2.701657e+04           5.144071       4.701608e+04
+1         0.128404    1.142854e+03           0.206674       1.848436e+03
+2       187.893502    1.687485e+06         585.681547       5.272459e+06
+3         0.000000    0.000000e+00           0.000000       0.000000e+00
+4       414.155914    3.713974e+06         826.624554       7.425320e+06
+
+    percentDiffExchangeDiffBtc  percentDiffExchangeDiffUsd  \
+0                   -37.732414                  -53.544924
+1                 -1017.120786                -1082.665604
+2                    14.589978                   31.029558
+3                          NaN                         NaN
+4                 -2890.486742               -15372.926874
+
+    percentDiffExchangeVolumeBtc  percentDiffExchangeVolumeUsd ticker
+0                      -6.591777                      8.992750    STX
+1                    -718.046381                   -670.219511    STU
+2                      10.075328                     22.742302    EDG
+3                            NaN                           NaN    PGL
+4                      41.544110                     48.203277    TNT
+
+```
+
+### Social Volume Projects
+
+Fetch a list of slugs for which there is social volume data.
+
+```python
+
+# The "/" at the end of the first argument is important. If you omit it you'll get an error.
+prices = san.get("social_volume_projects/")
```

Comment:
The trailing slash looks weird to me. What do you think?

Reply:
Yes, we should make it work without the trailing slash. I think it will be better